### PR TITLE
Feature/add map

### PR DESCRIPTION
**Pull request by NicoleLatifi**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

_Added Map Component including a map SVG file. From my understanding, just adding an image with an SVG file will not allow the SVG to be dynamic so it needs to be structured this way._

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

Open in browser and view the map displaying.

# Message for the reviewer:

Does this file structure make sense?

# Checklist:
- [ ] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/app/map/map.component.svg [hunk 1]
```diff
@@ -0,0 +1,5119 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (Leasw144) [inline src/app/map/map.component.svg:1]: not sure if we'll want to just add a img tag here and import the svg from assets but this is a good stepping stone!

**Review by Leasw144 — APPROVED** [review]
Thanks for getting the head start on this!

File: src/app/map/map.component.svg (hunk 1 above)
Comment (JoshSevy) [reply src/app/map/map.component.svg:1]: I think this is **great**, within this file we can actually add styling, change the sizing and zoom. The `svg` I am working on has a lot less break points so hopefully wont be so overwhelming. 

Great work ,this is a great start.

**Review by JoshSevy — APPROVED** [review]
Great work, @NicoleLatifi. This is such a great start to figuring out the map portion of the project.

File: src/app/map/map.component.svg [hunk 2]
```diff
@@ -0,0 +1,5119 @@
...
+	<line class="st0" x1="1014.86" y1="290.44" x2="1014.86" y2="476"/>
+	<line class="st0" x1="818.86" y1="280" x2="818.86" y2="476"/>
+	<line class="st0" x1="835.4" y1="280" x2="818.86" y2="280"/>
+	<line class="st0" x1="865.94" y1="280" x2="1014.86" y2="280"/>
+	<line class="st0" x1="1014.86" y1="290.44" x2="1014.86" y2="280"/>
+	<line class="st0" x1="710.47" y1="728" x2="711.51" y2="518"/>
+	<line class="st0" x1="1202.86" y1="29" x2="1234.32" y2="29"/>
+	<line class="st0" x1="1238.3" y1="728.87" x2="1234.32" y2="29"/>
+</g>
+<g id="Layer_2" class="st9">
```
Comment (NicoleLatifi) [inline src/app/map/map.component.svg:192]: I'm not sure if everything within this <g></g> needs to be here. When I comment it out it seems to still work.

**Pull request merged**